docs/react-integration.md

```diff
@@ -131,6 +131,10 @@ function App() {
 }
 ```
 
+`useQuery` also supports a "disabled" parameter: if set to `false` the query won't
+be sent. This is useful when you have dependent queries that rely on the results
```

Comment:
I find that using `sent` is difficult to understand when we talk about a `query`. In my head, `sent` for a query, is the action to fetch the server. 

Here, we do not only fetch the server, but we do not execute anything. Maybe we should use `executed` instead of `sent`?

Reply:
J'ai pas compris exactement ce que tu voulais dire, j'ai essayé de clarifier en

```patch
-be sent. This is useful when you have dependent queries that rely on the results
-of another query to be sent.
+be executed. This is useful when you have dependent queries that rely on the results
+of another query to have been fetched.
```

Qu'est ce que tu en penses ?